tests/data/dataset_readers/SQUAD_SAMPLE_FILE.json

```diff
@@ -0,0 +1,34 @@
+{
```

Comment:
Also, I would probably put this file in a separate directory from python code.  Not sure what to call it - `tests/resources/`?  Not sure what is typically done in python.  Another alternative would be to just have this json in the test itself, write it to a file in the temp test directory, then read it from that file.  I'd probably prefer that alternative, actually.  That would make the test file itself much more self-contained.